_posts/2023-01-01-seraphis-jamtis-developer-opportunities.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Developer Opportunities at the Monero Project
+summary: Join the Monero dev community to implement the two brand-new technologies Seraphis and Jamtis 
+tags: [community, development, seraphis, jamtis, ccs, magic]
+author: René Brunner (rbrunner7)
+---
+
+### TL;DR
+
+Monero is a cryptocurrency started in 2014. It's often called a *privacy coin*: Transaction senders, receivers and amounts are all not visible to outsiders on the blockchain, in stark contrast to most other cryptocurrencies, including Bitcoin. However, many proponents do not primarily see privacy as an end in itself but something that is needed for Monero for being able to function as a true electronic version of cash. For an ELI5 check e.g. [What is Monero](https://www.getmonero.org/get-started/what-is-monero/).
+
+The Monero dev community is currently implementing two brand-new technologies called *Seraphis* and *Jamtis* to improve privacy further and add functionality. According to some still very rough estimates those may be ready to bring into service with a hardfork in 2 years' time. Quite a lot of new code needs to get implemented, and a lot of existing code modified and adjusted; it's probably the biggest single coding related project for the currency ever.
+
+Does that sound like something you may want to work on? Can you write sophisticated C++ code, or are you able to learn to do so "on the job", and ready to work as a member of an open source dev team? Then please have a closer look.
+
+There is no company behind Monero to manage dev work; it's an open source project as true as they get nowadays. There are no VC companies involved in financing either that could force the project into directions that benefit themselves more than the Monero users. "Meritocracy" may be pretty grand and a bit overused as a concept, but if your work has merit you will get your code merged into software that is used to manage a cryptocurrency with a market valuation in the billions of USD.
+
+### Project Background
+
+Monero came into existence 2014 when a group of 7 devs forked the code of a cryptocurrency called *Bytecoin* to start a new, independent blockchain with a fresh genesis block. Its starting point was therefore a moderately large C++ codebase built from scratch since maybe 2011 to implement a new privacy coin technology called *CryptoNote* for the first time.
```

Comment:
```suggestion
Monero came into existence in 2014 when a group of 7 devs forked the code of a cryptocurrency called *Bytecoin* to start a new, independent blockchain with a fresh genesis block. Its starting point was therefore a moderately large C++ codebase built from scratch since maybe 2011 to implement a new privacy coin technology called *CryptoNote* for the first time.
```

Reply:
Corrected.